### DISCUSSION TITLE
How To Set Kernel Parameter fips=1

### DISCUSSION THREAD

**Discussion by srguglielmo** [bottlerocket-os/bottlerocket#3484] [Q&A] [answered]
Hello! I am looking to pass the parameter `fips=1` to the kernel as mentioned in https://github.com/bottlerocket-os/bottlerocket/issues/1667. This will then allow my containers to bind-mount `/etc/system-fips` from the host (Bottlerocket) into the container so that FIPS mode in the container can be properly validated. What is the best way to do this? I'm not sure if this can be done in an AWS Launch Template, or if the bootloader (grub?) needs to be edited, or something else. Is there any guidance on how to pass kernel parameters? Thank you!

**Comment by jpculp** [comment, marked as answer]
Thank you for reaching out! As far as I am aware it is not possible with Bottlerocket out-of-the-box until after some of the work described in #1667 is done.

**Reply by srguglielmo** [reply]
Thanks for your reply. I'll keep an eye on #1667.
